apps/meteor/app/livechat/imports/server/rest/upload.ts

```diff
@@ -9,24 +9,14 @@ import { FileUpload } from '../../../../file-upload/server';
 import { API } from '../../../../api/server';
 import { getUploadFormData } from '../../../../api/server/lib/getUploadFormData';
 
-let maxFileSize;
-
-settings.watch('FileUpload_MaxFileSize', async function (value) {
-	try {
-		maxFileSize = parseInt(value);
-	} catch (e) {
-		maxFileSize = await Settings.findOneById('FileUpload_MaxFileSize').packageValue;
-	}
-});
-
 API.v1.addRoute('livechat/upload/:rid', {
 	async post() {
 		if (!this.request.headers['x-visitor-token']) {
 			return API.v1.unauthorized();
 		}
 
```

Comment:
```suggestion
		if (typeof visitorToken !== 'string') {
			return API.v1.failure('Invalid visitor token');
		}
		const visitor = await LivechatVisitors.getVisitorByToken(visitorToken as string, {});
```

Enforcing the type check here to prevent any injection

Reply:
In which cases the type of the header value will be something different from string, since we're not parsing it? 👀